src/rustup-dist/build.rs

```diff
@@ -1,15 +1,6 @@
 use std::env;
-use std::fs::File;
-use std::io::Write;
-use std::path::PathBuf;
 
 fn main() {
-    let out_dir = PathBuf::from(env::var_os("OUT_DIR").unwrap());
     let target = env::var("TARGET").unwrap();
-
-    File::create(out_dir.join("target.txt"))
-        .unwrap()
-        .write_all(target.as_bytes())
-        .unwrap();
-    println!("cargo:rerun-if-changed=build.rs");
```

Comment:
Looks like my comment was lost: I thought we would need to keep this directive in order to avoid unnecessary rebuilds?

Reply:
I was torn about whether to assume this was still necessary or whether it was legacy cruft.  Looks like I went with the latter assumption.  I'll recover it and we can dump it another day if need be.